boilerplate/App/Config/DebugConfig.js

```diff
@@ -4,5 +4,7 @@ export default {
   yellowBox: __DEV__,
   reduxLogging: __DEV__,
   includeExamples: __DEV__,
-  useReactotron: __DEV__
+  useReactotron: __DEV__,
+  launchStorybook: __DEV__
 }
+
```

Comment:
From `npm run lint`
`DebugConfig.js:10:1: Too many blank lines at the end of file. Max of 0 allowed.`
Would you mind deleting the line and pushing? The the build server will run the integration tests.

Reply:
Hi. Lint is now ok but integration tests are failing.